modules/tinymce/src/themes/silver/main/ts/backstage/Backstage.ts

```diff
@@ -61,7 +62,8 @@ const init = (sink: AlloyComponent, editor: Editor, lazyAnchorbar: () => AlloyCo
       providers: {
         icons: () => editor.ui.registry.getAll().icons,
         menuItems: () => editor.ui.registry.getAll().menuItems,
-        translate: I18n.translate
+        translate: I18n.translate,
+        isReadonly: () => editor.readonly
```

Comment:
This possibly should be `isReadonly: editor.mode.isReadonly` since it's what's used elsewhere?